spec/factories/user.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :user do
+    email 'john.smith@example.com'
```

Comment:
Since email is unique, you need sequence here (otherwise you can't create two users without specifying emails)
`sequence(:email) { |n| "user#{n}@example.com" }`